packages/dev/core/src/Materials/GreasedLine/greasedLinePluginMaterialShadersGLSL.ts

```diff
@@ -0,0 +1,161 @@
+import type { Nullable } from "../../types";
+import { GreasedLineMeshColorMode } from "./greasedLineMaterialInterfaces";
+
+/**
+ * Returns GLSL custom shader code
+ * @param shaderType vertex or fragment
+ * @param cameraFacing is in camera facing mode?
+ * @returns GLSL custom shader code
```

Comment:
You should add @internal, as there's no reason why end users should call the function themselves (same for WGSL code).